coord/src/rmr/rmr.c

```diff
@@ -614,6 +617,15 @@ void iterManualNextCb(void *p) {
   }
 }
 
+void iterManualNextReadCb(void *p) {
+  MRIterator *it = p;
+  // Unsafely changing the flag is ok since we run this callback from the RQ,
+  // i.e no race conditions with the shards.
+  it->ctx.freeFlag = false; // Give the writers their reference back
```

Comment:
Fixing the release mechanism in a following PR?

Reply:
yes